src/Ocis.php

```diff
@@ -306,6 +309,29 @@ private function getServiceUrlFromWebfinger(string $webfingerUrl): string
         }
         throw new InvalidResponseException('invalid webfinger response');
     }
+
+    /**
+     * saves current oCIS version in semantic versioning format ( e.g. "5.0.5" )
+     *
+     * @return void
+     * @throws InvalidResponseException
+     */
+    private function getOcisVersion(): void
```

Comment:
if it set ocis version . how about calling setOcisVersion rather than getOcisVersion

Reply:
naming is hard - `setOcisVersion` sounds like it could be called, and it will set ocis to some version (of course, it does not take any parameter, so a developer will see that you can't specify a version and force ocis to change to run that version)

Maybe it will be clearer if this method returns the current ocis version string. The code in the constructor will look more "normal". It will have something like:
`$this->ocisVersion = $this->getOcisVersion();`